.erb-lint.yml

```diff
@@ -0,0 +1,11 @@
+---
+exclude:
+  - '**/.github/**/*'
+  - '**/.stotybook/**/*'
+  - '**/node_modules/**/*'
```

Comment:
I think these are good excludes in general.

Reply:
Thanks for the feedback @wwahammy.